app/src/main/java/org/mozilla/tv/firefox/pocket/PocketViewModel.kt

```diff
@@ -34,7 +34,8 @@ class PocketViewModel(private val pocketVideoRepo: PocketVideoRepo, pocketRepoCa
             val title: String,
             val url: String,
             val thumbnailURL: String,
-            val popularitySortId: Int
+            val popularitySortId: Int,
+            val authors: JSONObject
```

Comment:
I think this field should be a `String`.

See comment in PocketVideoAdapter

Reply:
This should already be a string in the last commit.